build.gradle.kts

```diff
@@ -79,3 +82,20 @@ dependencies {
 
     compile("com.google.protobuf:protobuf-java:2.5.0")
 }
+
+val intellijRepoChannel: String by project
+val publishPlugin: PublishTask by tasks
+publishPlugin {
+    username(System.getenv("CONTAINER_TOOLS_REPO_USERNAME"))
+    password(System.getenv("CONTAINER_TOOLS_REPO_PASSWORD"))
+    channels(intellijRepoChannel)
+}
+
+release {
+    tagTemplate = "v\${version}"
+
+    val git: GitAdapter.GitConfig = getProperty("git") as GitAdapter.GitConfig
+    git.requireBranch = "/^release_v\\d+.*$/"
+}
+
+inline operator fun <T : Task> T.invoke(a: T.() -> Unit): T = apply(a)
```

Comment:
Interesting, we convert apply to direct call for Task?

Reply:
Yeah. Makes the task definitions a bit cleaner